test/testdata/MovementsTestData.scala

```diff
@@ -47,9 +47,9 @@ object MovementsTestData {
   val date = Date(Some(10), Some(8), Some(2018))
   val departureDetails = DepartureDetails(date)
 
-  def arrivalDepartureTimes(movementType: String): JsValue = movementType match {
-    case "EAL" => Json.toJson(ArrivalDetails(date, Time(Some("13"), Some("34"))))
-    case _     => Json.toJson(DepartureDetails(date))
+  def arrivalDepartureTimes(movementType: Choice): JsValue = movementType match {
+    case Arrival => Json.toJson(ArrivalDetails(date, Time(Some("13"), Some("34"))))
+    case _       => Json.toJson(DepartureDetails(date))
```

Comment:
This methods looks

Reply:
MovementTestData, moreover it's old implementation